index.js

```diff
@@ -1,165 +1,182 @@
 #!/usr/bin/env node
 
-require('./src/ImageSequencer');
-sequencer = ImageSequencer({ui: false});
-var Spinner = require('ora')
+require("./src/ImageSequencer");
```

Comment:
Why change all the quotes?

1. It adds noise to the change.
2. Single quotes are usually preferred anyway.

Reply:
Oh I get this, this is my ide actually😅

Reply:
I'll get this